travel_journey/tests/test_journey.py

```diff
@@ -194,3 +197,11 @@ def test_inv_estimate_date(self):
             cr, uid, journey_id,
             field_name='date_stop', val='2014-03-10',
             arg=None, context=context)
+
+    def test_get_passenger(self):
+        cr, uid, vals, context = self.cr, self.uid, self.vals, self.context
+        journey_id = self.journey_model.create(cr, uid, vals, context=context)
+        journey_rec = self.journey_model.browse(
+            self.cr, self.uid, journey_id, context=self.context)
+        with self.assertRaises(AttributeError):
+            self._get_passenger(journey_rec)
```

Comment:
Why not just call `travel_journey_report._get_passenger` here and remove the multiple inheritance